src/edges_analysis/const.py

```diff
@@ -3,7 +3,7 @@
 from astropy import units as apu
 
 edges_lat_deg = -26.714778
-edges_lon_deg = 116.605528
+edges_lon_deg = 116.5 #116.605528
```

Comment:
I don't think this is the right thing to do -- we should use the correct value, not Alan's, by default. It just needs to be an option.

Reply:
Ah, I see. This is blocking the merge. Let me see if I can add this as an option